Cargo.toml

```diff
@@ -26,7 +26,7 @@ r2d2_sqlite = "0.1.*"
 rocket = "0.2.*"
 rocket_codegen = "0.2.*"
 rocket_contrib = "0.2.*"
-rusqlite = "0.10.*"
+rusqlite = "0.11.*"
```

Comment:
Thinking about it we can remove this dependency because r2d2_sqlite already requires it. What do you think ?